src/Microsoft.Data/BaseColumn.BinaryOperations.cs

```diff
@@ -72,8 +72,7 @@ public virtual BaseColumn And(BaseColumn column, bool inPlace = false)
             throw new NotImplementedException();
         }
 
-        public virtual BaseColumn And<T>(T value, bool inPlace = false)
-            where T : unmanaged
+        public virtual BaseColumn And(bool value, bool inPlace = false)
```

Comment:
Can I only get back a `PrimitiveColumn<bool>` from this method?

Reply:
Yes, it only returns a `PrimitiveColumn<bool>`. You are thinking about marking the return type PrimitiveColumn<bool> correct?

Reply:
I think that makes sense. Especially since we are changing the input type to `bool`.